### DISCUSSION TITLE
LSPatch fork compatible with earlier versions of Android

### DISCUSSION THREAD

**Discussion by Bardock88** [JingMatrix/ChromeXt#122] [General]
🙋‍♂️Hello, is there any way you can change the minimum API installation requirement for LSPatch from Android 9.0(API28) to Android 8.0(API26)?
Because I have this version of Android and like ChromeXt it is possible to install it on Android 5.0 (API21) and I have a lot of bookmarks and I want to export them from Chrome for a backup and I think it would help a lot of people with this change.

**Comment by JingMatrix** [comment]
Nope. You need root permission to achieve what you need, because you are exporting data from an installed app.
For LSPatch, you must uninstall the old app before using it.

**Reply by Bardock88** [reply]
> Nope. You need root permission to achieve what you need, because you are exporting data from an installed app. For LSPatch, you must uninstall the old app before using it.

Sorry I didn't express myself well, LSPatch is a non-root Xposed framework that extends from LSPosed, right?
So I want to use it together with ChromeXt to export my Chrome bookmarks without having to use 'BigG' synchronization and without having to be a Super User (Root), of course if it's possible to do the feat I'm thinking about 😐.
In addition, both versions of LSPatch (original author and this fork) need Android 9.0 to install and I really want to use it on my phone that runs Android 8.0, with the ChromeXt module that extends the functionality of Chromium browsers, with the possibility can even export bookmarks, which is why I asked if you could change the minimum requirement from Android 9.0 to an earlier version 😐🙏.

**Reply by JingMatrix** [reply]
To install the patched version of Chrome, you need to uninstall __the Chrome application installed on your phone__, which contains the bookmark data.
Once the `Chrome` application is unintalled, you have no way to recover the bookmark data, and so impossible to export them.
